picard/util/__init__.py

```diff
@@ -383,21 +383,19 @@ def album_artist_from_path(filename, album, artist):
     return album, artist
 
 
-def build_qurl(host, port=80, path=None, mblogin=False, queryargs=None):
+def build_qurl(host, port=80, path=None, queryargs=None):
     """
     Builds and returns a QUrl object from `host`, `port` and `path` and
     automatically enables HTTPS if necessary.
 
-    Setting `mblogin` to True forces HTTPS on MusicBrainz' servers.
-
     Encoded query arguments can be provided in `queryargs`, a
     dictionary mapping field names to values.
     """
     url = QtCore.QUrl()
     url.setHost(host)
     url.setPort(port)
-    if (# Login is required and we're contacting an MB server
-        (mblogin and host in MUSICBRAINZ_SERVERS and port == 80) or
+    if (# We're contacting a MusicBrainz server
+        (host in MUSICBRAINZ_SERVERS and port == 80) or
```

Comment:
Does it make sense to check for `port == 80`? Maybe it should just check for `host in MUSICBRAINZ_SERVERS` to prevent typos etc.?